.github/workflows/nextjs.yml

```diff
@@ -0,0 +1,98 @@
+# Sample workflow for building and deploying a Next.js site to GitHub Pages
+#
+# To get started with Next.js see: https://nextjs.org/docs/getting-started
+#
+name: Deploy Next.js site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["dev"]
```

Comment:
Should probably be `main` here, right?

Reply:
here nestor asked me to use dev

Reply:
Alright, I leave this decision to you. 

My only comments on that are the following

- For the "final" project I would want this project to be in Github flow style with `main` branch. I think a traditional flow with `develop`, `staging` and `production` branches is waaaay to bulky given how small the project is. But no objections at all to use it temporarily, if this helps your development
- My only requirement for development phase is, that I would like to have an easy way to take a look at the latest state of development via the deployed version.